src/Data/Expenses/Parse/Megaparsec/Document.hs

```diff
@@ -2,46 +2,59 @@ module Data.Expenses.Parse.Megaparsec.Document
   ( modelFromAst
   , parseExpensesFile
   , withFile
-  ) where
-
-import Control.Monad (forM_, void)
-
-import Data.Void (Void)
-
-import Data.Either (Either(..), partitionEithers)
-
-import qualified Data.Time.Calendar.Compat as DT
-
-import Text.Megaparsec
-  ( ParseError
-  , anySingle
-  , between
-  , choice
-  , eof
-  , errorBundlePretty
-  , hidden
-  , manyTill
-  , parseErrorPretty
-  , runParser
-  , sepEndBy
-  , skipMany
-  , someTill
-  , try
-  , withRecovery
-  , (<?>)
-  , (<|>)
   )
-import Text.Megaparsec.Char (eol, spaceChar)
-import qualified Text.Megaparsec.Char.Lexer as L
-
-import Data.Expenses.Parse.Megaparsec.Entry
-import Data.Expenses.Parse.Megaparsec.Types
-  (AST(..), LineDirective(..), Parser, RawLineDirective)
-import qualified Data.Expenses.Parse.Megaparsec.DateDirective as PD
-import qualified Data.Expenses.Parse.Megaparsec.ExpenseDirective as PE
-import Data.Expenses.Expense
-  (nextDate)
-import Data.Expenses.Types (Model, modelFromEntries)
+where
+
+import           Control.Monad                  ( forM_
+                                                , void
+                                                )
+
+import           Data.Void                      ( Void )
+
+import           Data.Either                    ( Either(..)
+                                                , partitionEithers
+                                                )
+
+import qualified Data.Time.Calendar.Compat     as DT
+
+import           Text.Megaparsec                ( ParseError
+                                                , anySingle
+                                                , between
+                                                , choice
+                                                , eof
+                                                , errorBundlePretty
+                                                , hidden
+                                                , manyTill
+                                                , parseErrorPretty
+                                                , runParser
+                                                , sepEndBy
+                                                , skipMany
+                                                , someTill
+                                                , try
+                                                , withRecovery
+                                                , (<?>)
+                                                , (<|>)
+                                                )
```

Comment:
This kindof thing does strike me as a bit awkward. -- Though I don't know if that's because I should be using a qualified import instead.